.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
npm doesn't seem like the right option. Is there any other one?

Reply:
There should be maven one, I keep getting npm and mvn mixed up